The Lonely Moon/Playfield/Playfield.gd

```diff
@@ -217,9 +217,6 @@ func create_debris(pos, vel, amount, radius, impluse):
 
 
 func craft_collision(craft1, craft2):
-    if craft1.type == "debris" and craft2.type == "debris":
```

Comment:
Won't removing this cause debris-debris collisions to generate even more debris?

Reply:
I think its fine.
Ben know's best .

Reply:
It won't because the checks on line 223 and 229 already do the same thing. But if it helps the code read easier, we can keep it in.